packages/vaadin-overlay/test/position-mixin-listeners.test.js

```diff
@@ -0,0 +1,217 @@
+import { expect } from '@esm-bundle/chai';
+import { fire, fixtureSync, nextFrame } from '@vaadin/testing-helpers';
+import sinon from 'sinon';
+import '../vaadin-overlay.js';
```

Comment:
nit: this import is not needed because the class is already imported below.

Reply:
Done in two places.